desktop-exporter/types.go

```diff
@@ -0,0 +1,71 @@
+package desktopexporter
+
+import (
+	"time"
+)
+
+type RecentSummaries struct {
+	TraceSummaries []TraceSummary `json:"traceSummaries"`
+}
+
+type TraceSummary struct {
+	TraceID   string `json:"traceID"`
+	SpanCount uint32 `json:"spanCount"`
+}
+
+type TraceData struct {
+	Spans      []SpanData `json:"spans"`
+	DurationMS int64      `json:"duration"`
```

Comment:
In general I find it very useful to include the unit in the name for durations when you don't have a type system to convey that information

```suggestion
	DurationMS int64      `json:"durationMS"`
```

Alternatively you could send along the start and end timestamps and calculate the duration on the client 🤔